recipes/libnl/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "libnl" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://www.infradead.org/~tgr/libnl/files/libnl-{{ version }}.tar.gz
+  sha256: 4f80c21fe5bbcdde6e72b59b4f98306063a41421f909887c34e58d93e746d063
+
+build:
+  skip: True  # [not linux]
+  number: 0
+  run_exports:
+    # https://abi-laboratory.pro/?view=timeline&l=libnl
+    - {{ pin_subpackage('libnl', 'x.x.x') }}
+  script:
+    - ./configure --prefix=$PREFIX
+    - make -j$CPU_COUNT
+    - make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libnl.so
+    - test -f $PREFIX/include/netlink/types.h
+
+about:
+  home: https://www.infradead.org/~tgr/libnl/
+  license: LGPL 2.1
+  license_file: COPYING
+  license_family: GPL
+  summary: "Netlink Library Suite"
+  description: |
+    The libnl suite is a collection of libraries providing APIs to netlink
+    protocol based Linux kernel interfaces. Netlink is a IPC mechanism primarly
+    between the kernel and user space processes. It was designed to be a more
+    flexible successor to ioctl to provide mainly networking related kernel
+    configuration and monitoring interfaces.
+  dev_url: github.com/tgraf/libnl-1.1-stable.git
```

Do you plan to update this to version 3 after version 1 has been deployed?
```suggestion
  dev_url: https://github.com/thom311/libnl
```

---

Will change after updating it to 3.x the one provided you doesn't have tags for 1.x

---

In that case the URL should be:
```suggestion
  dev_url: https://github.com/tgraf/libnl-1.1-stable
```